requirements.txt

```diff
@@ -0,0 +1 @@
+-e .
```

Comment:
I'll add that, but I believe rtfd still needs a requirements.txt in order to build the documentation.